### DISCUSSION TITLE
6.2.0.5

### DISCUSSION THREAD

**Discussion by Ghost-chu** [QuickShop-Community/QuickShop-Hikari#1607] [Announcements]
## New Features

1. 1.21 support

## Bug Fixes

1. Fixed `/qs clean` may delete all stores
2. Fixed `/qs cleanghost` may delete all stores

<hr /><em>This discussion was created from the release <a href='https://github.com/Ghost-chu/QuickShop-Hikari/releases/tag/6.2.0.5'>6.2.0.5</a>.</em>

**Comment by keijodputt** [comment]
Potato's Reremake added Folia compatibility on their 5.1.2.5 snapshot; do you have any plans to add it as well?

**Reply by Ghost-chu** [reply]
> Potato's Reremake added Folia compatibility on their 5.1.2.5 snapshot; do you have any plans to add it as well?

You are misinformed, Reremake has never had Folia support, includes snapshot, I'm also Reremake's dev.  
Hikari has no plans to support Folia at this time.

**Reply by keijodputt** [reply]
Indeed, I was misinformed. EarthMC has a custom reremake running on Folia because of Warriorrr adapting it for them.

**Reply by GoodrichDev** [reply]
There should definitely be some Folia support but hey, that's what forking is fork! *fork*
